Submissions/PratyushPareek/Task2/task2.py

```diff
@@ -0,0 +1,13 @@
+import shutil
+import os
+
+s = '..\\...\\Monty_Python_Class\\Task_2\\My_Files\\'
```

Comment:
Use os.path.join() to join path

Reply:
![Capture](https://user-images.githubusercontent.com/58395391/73018618-06bf7700-3e48-11ea-9b1f-2b4a6af707c0.PNG)
I've used it at all the places it can be used :/
Line 4 is just creating a variable to store the location to the folder. 
All the times I have joined two or more parts of an address, I have used os.path.join() to do it

Reply:
Use there also or ask user to give path but don't use concrete path using \\ as other operating system users cannot run the code